src/main/java/bio/terra/cli/utils/mount/MountController.java

```diff
@@ -0,0 +1,285 @@
+package bio.terra.cli.utils.mount;
+
+import bio.terra.cli.app.utils.LocalProcessLauncher;
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.businessobject.Resource;
+import bio.terra.cli.businessobject.Workspace;
+import bio.terra.cli.businessobject.resource.GcsBucket;
+import bio.terra.cli.businessobject.resource.GcsObject;
+import bio.terra.cli.exception.SystemException;
+import bio.terra.cli.exception.UserActionableException;
+import bio.terra.cli.utils.mount.handlers.BaseMountHandler;
+import bio.terra.cli.utils.mount.handlers.GcsFuseMountHandler;
+import bio.terra.workspace.model.Folder;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.UUID;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+/** This class provides utility methods for mounting and unmount workspace resources */
+public abstract class MountController {
+
+  // Directory to mount workspace resources under
+  private static final Path WORKSPACE_DIR = Paths.get(System.getProperty("user.home"), "workspace");
```

Comment:
Use `Context.getContextDir()` as the base to respect the context dir override env var. That will also add `.terra` to the path so we're not working directly in `~/workspace`

Reply:
Got it. I'm a little wary about writing to `~/workspace` which doesn't have any indication it's associated with Terra, but if we're generally expecting people to use this in notebooks rather than locally, that's less of a concern